posts/rse_at_eth/index.qmd

```diff
@@ -0,0 +1,62 @@
+---
+title: "RSE@ETH - The Force awakens"
+description: "Inspired by the UK RSE chapter, the first in-person event of the Research Software Engineering (RSE) community at ETH Zurich took place on the 1st of February. With the aim of getting to know the various RSE interest groups represented at ETH, the participants were able to present themselves and their causes in rapid succession and discuss the future development of RSE@ETH."
```

Comment:
@minnaheim : chapter of what? Many of our readers if not most are economists at this stage. Hence they do not necessarily know the how RSE is organized. Minor remark: The blog listing and previous blog posts use February 1, 2024 as a date format. I'd stay consistent here. 


I suggest something a long the lines of:

Inspired by the UK Society of Research Software Engineering (RSE), the first in-person event of the newly founded RSE at ETH took place on February 1. A lightning talk format gave a broad array of interested researchers the chance to present themselves  and software engineering work at ETH and discuss the future development of RSE@ETH.